experimental/dds/tree2/src/feature-libraries/editable-tree/editableTreeBinder.ts

```diff
@@ -305,18 +326,20 @@ abstract class AbstractPathVisitor implements PathVisitor {
 	public abstract onInsert(path: UpPath, content: ProtoNodes): void;
 	public registerListener(
 		contextType: BindingContextType,
-		trees: BindTree[],
+		policies: BindPolicy[],
 		listener: Listener,
 	): () => void {
 		const contextRoots = getOrCreate(this.registeredListeners, contextType, () => new Map());
-		trees.forEach((tree) => {
+		policies.forEach((policy) => {
+			const tree = policy.bindTree;
 			const currentRoot = this.findRoot(contextType, tree.field);
 			if (currentRoot === undefined) {
 				const newRoot: CallTree = {
 					field: tree.field,
 					index: tree.index,
 					listeners: new Set(),
 					children: new Map(),
+					matchPolicy: policy.matchPolicy ?? "path",
```

Comment:
matchPolicy cannot be undefined so `??` default is unnecesary

Reply:
Done.